### PR TITLE
Add more job statistics

### PR DESCRIPTION
**Pull request by AFornio**
A couple things added:

-Change Duration format
-Add new column to to catch how long the job was in the queue BEFORE it started
-Add new column to see how many were in the queue when the job was queued

### CODE REVIEW

File: lib/resque/plugins/job_stats/enqueued.rb [hunk 1]
```diff
@@ -26,6 +26,28 @@ def after_enqueue_job_stats_enqueued(*args)
...
+        def number_of_jobs_in_queue
+          Resque.size(self.queue)
+        end
+
+        def been_in_the_queue_for(enqueued_since)
+          time_diff(Time.now, enqueued_since)
+        end
+
+        private
+        def time_diff(start_time, end_time)
```
Comment (sncalvo) [inline lib/resque/plugins/job_stats/enqueued.rb:39]: Missing space before `def time_diff`

File: lib/resque/plugins/job_stats/history.rb [hunk 2]
```diff
@@ -18,14 +18,19 @@ def around_perform_job_stats_history(*args)
...
           begin
             yield
             duration = Time.now - start
-            push_history "success" => true, "args" => args, "run_at" => start, "duration" => duration
+            push_history "success" => true, "args" => args, "run_at" => start, "duration" => duration, "number_of_jobs_in_queue" => number_of_jobs_in_queue, "been_in_the_queue_for" => been_in_the_queue_for
           rescue Exception => e
             duration = Time.now - start
             exception = { "name" => e.to_s, "backtrace" => e.backtrace }
-            push_history "success" => false, "exception" => exception, "args" => args, "run_at" => start, "duration" => duration
+            push_history "success" => false, "exception" => exception, "args" => args, "run_at" => start, "duration" => duration, "number_of_jobs_in_queue" => number_of_jobs_in_queue, "been_in_the_queue_for" => been_in_the_queue_for
```
Comment (sncalvo) [inline lib/resque/plugins/job_stats/history.rb]: Line too long

File: lib/resque/plugins/job_stats/history.rb [hunk 3]
```diff
@@ -18,14 +18,19 @@ def around_perform_job_stats_history(*args)
...
+
+          # new - fork
+          number_of_jobs_in_queue = self.number_of_jobs_in_queue
+          been_in_the_queue_for = self.been_in_the_queue_for(args.last.to_time)
+
           begin
             yield
             duration = Time.now - start
-            push_history "success" => true, "args" => args, "run_at" => start, "duration" => duration
+            push_history "success" => true, "args" => args, "run_at" => start, "duration" => duration, "number_of_jobs_in_queue" => number_of_jobs_in_queue, "been_in_the_queue_for" => been_in_the_queue_for
```
Comment (sncalvo) [inline lib/resque/plugins/job_stats/history.rb]: Line too long

**Review by sncalvo — COMMENTED** [review]
Can we add rubocop and lint the files changed?

**Review by sncalvo — APPROVED** [review]

**Review by santiagogabrielcastillo — APPROVED** [review]

**Pull request merged**